cmd/wksctl/kubeconfig/kubeconfig.go

```diff
@@ -80,6 +81,15 @@ func kubeconfigRun(cmd *cobra.Command, args []string) error {
 	if kubeconfigOptions.gitURL == "" {
 		// Cluster and Machine manifests come from the local filesystem.
 		clusterPath, machinesPath = kubeconfigOptions.clusterManifestPath, kubeconfigOptions.machinesManifestPath
+		// Check for cluster.yaml and machines.yaml paths
+		if clusterPath == "cluster.yaml" && machinesPath == "machines.yaml" {
```

Comment:
[Simon/Sara]: as the user might have actually set these we could check that instead. Maybe this IsSet fn? from here: https://github.com/spf13/cobra/issues/453

```go
if cmd.Flags().Changed("machines") ...
```

Perhaps? (untested).